### DISCUSSION TITLE
Patchstorage application for PiPedal

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#245] [General]
I have applied to get PiPedal added to [PatchStorage](https://patchstorage.com/). This would allow online sharing of PiPedal presets. 

I would very much appreciate it if you could upvote the request to add PiPedal. To upvote (or comment), click here:

&nbsp;&nbsp; [https://patchstorage.com/requests/pipedal/](https://patchstorage.com/requests/pipedal/)

Hoping to also work with the site owner to improve packaging of the Raspberry Pi LV2 plugin downloads.  But first things first.

**Comment by rerdavies** [comment]
Oh, and!!! Once the PatchStorage page for Pipedal goes live, I will need at least 5(!) presets to kick off the platform. So if you could give some thought to which presets you have that you might like to share, I would be grateful.

I'll let you know as soon as the application is approved, so that you can upload them yourselves.

You will be able to upload both presets and banks. Media files (models, IR files, &c) are automatically included in presets and banks that you share, as long as you are using a current version of PiPedal. And are carefully merged with existing models and IR files when you upload them to PiPedal, in case you were wondering. When uploading presets and banks to PiPedal, media files will be compared with existing files, and presets will reference existing files if they are identical, or will will be modified to reference ... (1), .. (2) variants as required. 

Presets you upload should reference models and IR files in the new shared directory structure, and NOT reference legacy folders.  In case you missed the release announcement, plugins now use shared directories for models and impulse files; but files that were uploaded before the introduction of the new directory structures cause an extra "legacy" directory to show up when you open the property browser dialog. It sounds complicated, but I think you'll get it when you see it in action.Your existing presets will continue to work; but new users of PiPedal won't see the legacy directories at all.

Presets or banks that use plugins other than the default Pipedal plugins will be  fine, as long as you list which plugins your preset(s) use in the upload description. Although it would be really nice to get some presets that use TooB NAM or TooB ML.

**Reply by FoolHen** [reply]
Hoping that it is accepted 😄 I can share some of the presets that I currently use, which sound pretty close to the original songs.
